github/issues_comments.go

```diff
@@ -10,6 +10,9 @@ import (
 	"time"
 )
 
+// MaxCommentLen is the maximum length of a comment that github will accept without a validationg error.
+var MaxCommentLen = 65535
```

Comment:
Whoops, will make it 65536. I swear I double checked that.

I made it external so developers could update the library if github ever changed their limit. No, it does not need to be external.